src/Popover.jsx

```diff
@@ -200,4 +204,9 @@ Popover.propTypes = {
 	className: React.PropTypes.string,
 };
 
+Popover.defaultProps = {
+	alignLeft: false,
+	alignRight: false,
```

Comment:
also you don't really need to set defaults as they are bool's it will evaluate to false on the cx() checks